cmds/coredhcp-generator/main.go

```diff
@@ -52,6 +54,13 @@ func main() {
 		if pl == "" {
 			continue
 		}
+		if !strings.ContainsRune(pl, '/') {
+			// A bare name was specified, not a full import path.
+			// Coredhcp plugins aren't in the standard library, assume this is one of the builtin plugins
```

Comment:
this assumption is not flawless: technically your import path may have package names without slashes (e.g. `import mymodule` would work if you have `$GOPATH/src/mymodule`). However this is very unlikely and I think we can run the risk. Maybe just update the comment to reflect this?

Reply:
Oh, I didn't consider that indeed. Since the `-from` version doesn't do this expansion, at least there is an easy workaround if anyone needs it; I'll update the comment and leave it at that